src/style.css

```diff
@@ -1 +1,68 @@
-/* Styles go here */
+body {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", Arial, sans-serif;
```

Comment:
move font styles to html tag
also, body doesn't have initial paddings, resetting paddings is redundant here